corehq/apps/users/tests/test_user_data.py

```diff
@@ -430,3 +430,9 @@ def test_reset_locations_and_user_data_updated(self):
             'commcare_location_ids': f"{self.loc1.location_id} {self.loc2.location_id}",
             'commcare_primary_case_sharing_id': self.loc1.location_id,
         })
+
+    def test_change_data_provided_by_system_will_raise_user_data_error(self):
```

Comment:
nit: less verbosity along the lines of `test_changing_system_provided_data_raises_user_data_error`